app/locale/af/LC_MESSAGES/django.po

```diff
@@ -183,7 +183,8 @@ msgstr ""
 #, python-format
 msgid ""
```

Comment:
This entry is still here. Can you try rerunning tools/update_messages to see if this entry disappears?